app/components/AiChatBase.tsx

```diff
@@ -0,0 +1,156 @@
+'use client';
+
+import Image from 'next/image';
+import { PropsWithChildren, useEffect, useState } from 'react';
+import * as Dialog from '@radix-ui/react-dialog';
+import { cn, MarkdownRenderer, ScrollArea } from '@/app/components';
+import aiStarsSvg from '@/public/ai-stars.svg';
+import wizardSvg from '@/public/pixel-wizard.svg';
+import { Icon, IconButton, Input, Tag } from '@swapr/ui';
+import { twMerge } from 'tailwind-merge';
+import { FA_EVENTS } from '@/analytics';
+import { trackEvent } from 'fathom-client';
+
+interface AiBaseProps extends PropsWithChildren {
+  messages: any[];
+  trackOnClickEvents: string[];
+  isChatOpen?: boolean;
+}
+
+const MARKETING_LINK =
+  'https://swpr.notion.site/Presagio-AI-Predictor-Chatbot-Beta-Launch-f4ccdfa867e949d3badf10705b7c90aa';
+
```

Comment:
Check the comments on this file for further context on the main function

```suggestion
const DEFAULT_BORDER_AND_BG_COLOR = 'bg-transparent';

```